src/ontology/uberon-edit.obo

```diff
@@ -145130,18 +145125,11 @@ property_value: external_definition "Exocrine glands in animals which secrete sc
 
 [Term]
 id: UBERON:0011253
-name: gland of anal sac
-synonym: "anal gland" RELATED []
-synonym: "anal sac gland" EXACT []
-synonym: "castor sac" NARROW SENSU [NCBITaxon:29132]
-synonym: "musk gland" RELATED []
-xref: NCIT:C32068
-xref: SCTID:91610001
-xref: UMLS:C0162322 {source="ncithesaurus:Anal_Gland"}
-xref: Wikipedia:Anal_gland
-is_a: UBERON:0011252 ! scent gland
-intersection_of: UBERON:0002365 ! exocrine gland
-intersection_of: part_of UBERON:0008978 ! anal sac
+name: obsolete gland of anal sac
+comment: Obsoleted as it seems to refer to the same concept as 'gland of anal canal'.
+property_value: term_tracker_item "https://github.com/obophenotype/uberon/issues/3159" xsd:anyURI
+is_obsolete: true
+replaced_by: UBERON:0004760
```

Comment:
Please, consider to add 'gland of anal sac' as a synonym to 'gland of anal canal'.